code/modules/projectiles/guns/special/syringe_gun.dm

```diff
@@ -84,16 +86,30 @@
 	var/syringe_count = syringes.len
 	. += "[initial(icon_state)]_[syringe_count ? clamp(syringe_count, 1, initial(max_syringes)) : "empty"]"
 
+/obj/item/gun/syringe/medical
+	name = "medical syringe gun"
+	desc = "A spring loaded gun designed to fit syringes, used to incapacitate unruly patients from a distance."
+	icon_state = "medicalsyringegun"
+	lefthand_file = 'icons/mob/inhands/weapons/64x_guns_left.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/64x_guns_right.dmi'
+	inhand_icon_state = "medicalsyringegun"
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	worn_icon_state = null
+	w_class = WEIGHT_CLASS_BULKY
+	slot_flags = ITEM_SLOT_BACK
+	force = 6 //A little bit more robust in melee since it's larger
```

Comment:
```suggestion
	force = 6
```
This comment pertains to the PR, but not to the code.
People reading it in the future won't know what it is about.